__tests__/app.test.js

```diff
@@ -179,4 +179,66 @@ describe("/api/articles", () => {
       });
     });
   });
+  describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: responds with a 200 status and an array of comment objects of length 11 for article_id 1", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(11);
+        });
+    });
+    test("GET 200: returns an array of comment objects each with properties comment_id, body, votes, author, article_id, created_at, each of correct type and with article_id equal to the id passed", () => {
+      const article_id = 1;
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(comment.article_id).toBe(article_id);
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
```

Comment:
There are a few lines that could be cut from this test to clean up the code a little ✂️ 
- Line 195: `.expect(200)` is already covered on the test before this one
- Line 197: also covered by previous test
- Line 206: this is already covered by line 199 where you are asserting for the exact `article_id` above